.travis.yml

```diff
@@ -7,9 +7,8 @@ services:
 env:
   matrix:
     - PYTHON=2.7 PACKAGES="blosc futures faulthandler"
-    - PYTHON=3.4 COVERAGE=true DASK_EXPERIMENTAL_ZMQ=1
-    - PYTHON=3.5 CRICK=true ASYNCIO=true
-    - PYTHON=3.6 PACKAGES=blosc ASYNCIO=true
+    - PYTHON=3.5 COVERAGE=true PACKAGES=blosc CRICK=true DASK_EXPERIMENTAL_ZMQ=1
```

Comment:
I think it's value is in verifying that comms is extensible.  However now that `inproc` exists that may suffice.  I'm inclined to remove it just to reduce the codebase.